tests/Aspire.EndToEnd.Tests/README.md

```diff
@@ -0,0 +1,65 @@
+# Aspire.EndToEnd.Tests
+
+## TL;dr or How do I use this?
+
+### Steps to prepare for `out-of-repo` runs
```

Comment:
Correct me if I'm wrong, but all of these out-of-repo instructions are mainly for helix setup, right?

I assume most people checking this readme will be interested in learning about these tests, adding new ones, and running them locally (in-tree). Nothing wrong with the instructions you have, but might be good to add at the top a description of what the tests are for, briefly explain an overview of how they work (meaning how they launch testproject.apphost, etc) and how to add new tests. And then we can explain the out of tree and how to set it up.

Anyway, don't block on this feedback, but perhaps consider it for expanding this readme in the future.

Reply:
Oh, I see you go over this a bit bellow, sorry for missing it I originally thought the contents were the same as your gist. Anyway, as mentioned above, feel free to ignore.